models/order.model.js

```diff
@@ -3,7 +3,7 @@ const mongoose = require('mongoose')
 const orderSchema = mongoose.Schema({
     user_id: {
         type: mongoose.Schema.Types.ObjectId,
-        ref: 'User',
+        ref: 'user',
```

Comment:
lets have all ref key in Capital e.g User or Product.
also do well to update them in the model like mongoose.model('Product', productSchema)

Reply:
done